walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/EventTest.java

```diff
@@ -34,20 +34,21 @@
   private static final int TIME_0830 = TimeRange.getTimeInMinutes(8, 30);
   private static final int TIME_1330 = TimeRange.getTimeInMinutes(13, 30);
 
-  private static final Event EVENT_A = new Event(
-      "Event A", TimeRange.fromStartDuration(TIME_0830, DURATION_30_MINUTES), Arrays.asList());
-  private static final Event EVENT_B = new Event(
-      "Event B", TimeRange.fromStartDuration(TIME_1330, DURATION_60_MINUTES), Arrays.asList());
-
   @Test
-  public void orderEventByStartTime() {
-    Event[] expected = {EVENT_A, EVENT_B};
+  public void orderEventsByStartTime() {
+    // The late event should be added to the queue before the early event. This means the priority
+    // queue should re-order them
+    Event earlyEvent = new Event("Early Event",
+        TimeRange.fromStartDuration(TIME_0830, DURATION_30_MINUTES), Arrays.asList());
+    Event lateEvent = new Event(
+        "Late Event", TimeRange.fromStartDuration(TIME_1330, DURATION_60_MINUTES), Arrays.asList());
 
     PriorityQueue<Event> eventQueue = new PriorityQueue<>(Event.ORDER_BY_START_TIME);
-    eventQueue.add(EVENT_B);
-    eventQueue.add(EVENT_A);
+    eventQueue.add(lateEvent);
+    eventQueue.add(earlyEvent);
     Event[] actual = eventQueue.toArray(new Event[eventQueue.size()]);
```

Comment:
Ah, sorry I didn't catch this before: `toArray()` doesn't guarantee that it returns elements in a particular order, so this test could break. https://docs.oracle.com/javase/8/docs/api/java/util/PriorityQueue.html#toArray-T:A-

(In practice, it may not. But the API doesn't guarantee anything, and it's a very bad idea to rely on undocumented behavior.)